app/src/main/java/com/pwhs/quickmem/presentation/app/notification/component/TimeExtensions.kt

```
@@ -0,0 +1,28 @@
+        if (duration.toDays() > 14) {
+            onDelete()
+            "Expired"
+        } else {
+            when {
+                duration.toMinutes() < 1 -> "${duration.seconds}s"
+                duration.toHours() < 1 -> "${duration.toMinutes()}m"
+                duration.toDays() < 1 -> "${duration.toHours()}h"
+                else -> "${duration.toDays()}d"
+            }
+        }
+    } catch (e: Exception) {
```

detekt

[warning] The caught exception is swallowed. The original exception could be lost.

(detekt.exceptions.SwallowedException)